inquire/src/prompts/path_select/mod.rs

```diff
@@ -0,0 +1,365 @@
+    ///
+    /// When overriding the config in a prompt, NO_COLOR is no longer considered and your
+    /// config is treated as the only source of truth. If you want to customize colors
+    /// and still suport NO_COLOR, you will have to do this on your end.
+    pub fn with_render_config(mut self, render_config: RenderConfig<'a>) -> Self {
+        self.render_config = render_config;
+        self
+    }
+
+    /// Parses the provided behavioral and rendering options and prompts
+    /// the CLI user for input according to the defined rules.
+    ///
+    /// Returns the owned objects selected by the user.
+    ///>Error::OperationCanceled)`, but `Ok(None)`.
+    ///
+    /// Meanwhile, if the user does submit an answer, the method wraps the return
+    /// type with `Some`.
+    pub fn prompt_skippable(self) -> InquireResult<Option<Vec<PathEntry>>> {
+        match self.prompt() {
+            Ok(answer) => Ok(Some(answer)),
+            Err(InquireError::OperationCanceled) => Ok(None),
+            Err(err) => Err(err),
+        }
+    }
+
+    /// Parses the provided behavioral and rendering options and prompts
+    /// the CLI user for input according to the defined rules.
+    ///
+    /// Returns the owned objects selected by the user.
+    pub fn prompt(self) -> InquireResult<Vec<PathEntry>> {
+        self.raw_prompt()
+            .map(|op| op.into_iter().map(|o| o.value).collect())
+    }
+    /// Parses the provided behavioral and rendering options and prompts
+    /// the CLI user for input according to the defined rules.
+    ///
+    /// Returns a vector of [`ListOption`](crate::list_option::ListOption)s containing
+    /// the index of the selections and the owned objects selected by the user.
+    ///
+    /// This method is intended for flows where the user skipping/cancelling
+    /// the prompt - by pressing ESC - is considered normal behavior. In this case,
+    /// it does not return `Err(InquireError::OperationCanceled)`, but `Ok(None)`.
+    ///
+    /// Meanwhile, if the user does submit an answer, the method wraps the return
+    /// type with `Some`.
+    pub fn raw_prompt_skippable(self) -> InquireResult<Option<Vec<ListOption<PathEntry>>>> {
+        match self.raw_prompt() {
+            Ok(answer) => Ok(Some(answer)),
+            Err(InquireError::OperationCanceled) => Ok(None),
+            Err(err) => Err(err),
+        }
+    }
+
+    /// Parses the provided behavioral and rendering options and prompts
+    /// the CLI user for input according to the defined rules.
+    ///
+    /// Returns a [`ListOption`](crate::list_option::ListOption) containing
+    /// the index of the selection and the owned object selected by the user.
+    pub fn raw_prompt(self) -> InquireResult<Vec<ListOption<PathEntry>>> {
+        let terminal = get_default_terminal()?;
+        let mut backend = Backend::new(terminal, self.render_config)?;
+        self.prompt_with_backend(&mut backend)
+    }
```

Comment:
My reasoning for raw_prompts back then was that some devs might want to have the index so they could reference it to the vectors they created the prompt with. Since the options here will be dynamically retrieved, I don't see much value (or at all) in returning indexes here, so let's just remove these methods.

```suggestion
```